backend/search/searchapi.py

```diff
@@ -0,0 +1,38 @@
+import requests,json
+from flask import request
+
+def searchProduct(searchQuery):
+    '''
+    Sends a get request to nbxd search api and returns the response.
+    input agruments are search query,page number and sort key.
+    '''
+
+    pageNumber = request.args.get('page', default=1, type=int)
+    sortKey = request.args.get('sort', default="ftrd", type=str)
+    pageNumber=(pageNumber-1)*9
+    
+    if sortKey=='ftrd':
+        sortKey=""
+        
+    unbxdApi = f"https://search.unbxd.io/fb853e3332f2645fac9d71dc63e09ec1/demo-unbxd700181503576558/search?q={searchQuery}&rows=9&start={pageNumber}&sort={sortKey}"
+    response=requests.get(unbxdApi)
```

Comment:
check the status of the response before extracting the output: example `if response and response.status_code == 200`